### PR TITLE
Add desktop3 recipe

### PR DESCRIPTION
**Pull request by tadeu**
@eight04 I've added you as recipe maintainer because you're the author of the Python 3 port. Please check that you are OK becoming maintainer of the recipe.

### CODE REVIEW

**Comment by tadeu** [comment]
@jakirkham I'm skipping `osx` because `setup.py` have a bug when building there. I'm using `  skip: true  # [osx or py2k]`, but this isn't working and the build is still failing in OS X, do you know what is wrong?

**Comment by eight04** [comment]
Could it be letter case problem?
```
+ python setup.py install --single-version-externally-managed --record record.txt
Traceback (most recent call last):
  File "setup.py", line 21, in <module>
    long_description = read("readme.rst"),
  File "setup.py", line 11, in read
    with open(path.join(here, file), encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/staged-recipes/build_artefacts/work/desktop3-0.5.0/readme.rst'
```

**Comment by tadeu** [comment]
> Could it be letter case problem?

Most probably, because the file is named `README.rst`, but what is weird is that it is passing in Linux (Windows is case insensitive anyway). Maybe the fact that it is in a `zip` is messing the case in Linux but not in OSX.

File: recipes/desktop3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - desktop
+
+about:
+  home: https://github.com/eight04/pyDesktop3
+  license: GNU Lesser General Public License v3 or later (LGPLv3+)
+  summary: 'Simple desktop integration for Python.'
+  license_family: LGPL
```
Comment (tadeu) [inline recipes/desktop3/meta.yaml]: It isn't bundled in the {{.zip}} in PyPI

Comment (tadeu) [inline recipes/desktop3/meta.yaml]: I've made a PR to @eight04 to bundle the license files in the package.

**Comment by tadeu** [comment]
@scopatz Done

**Comment by scopatz** [comment]
thanks

**Pull request merged**